doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -302,3 +325,39 @@ The specific use case of the Google Fast Pair application is indicated by the ch
 Different use cases may require implementation of additional guidelines for your accessory firmware or specific configuration of your device model in the Google Nearby Devices console.
 These requirements typically help to improve user experience or security properties for the chosen use case.
 To learn more, see the official `Fast Pair`_ documentation.
+
+Applications and samples
+************************
+
+The following application and sample use the Fast Pair integration in |NCS|:
+
+* :ref:`nrf_desktop` application
+* :ref:`fast_pair_input_device` sample
+
+Library support
+***************
+
+The following |NCS| libraries support the Fast Pair integration:
+
+* :ref:`bt_fast_pair_readme` library is used to implement the Bluetooth® Low Energy (LE) GATT Service required for :ref:`ug_bt_fast_pair` with the |NCS|.
+* :ref:`bt_le_adv_prov_readme` provides the Fast Pair provider API.
```

Comment:
```suggestion
* :ref:`bt_le_adv_prov_readme` delivers the Fast Pair provider API.
```
not sure if that's the right substitution but just wanted to minimize providers

Reply:
May be we could use _generates_?

Reply:
sounds great!